vitest.config.mjs

```diff
@@ -265,7 +265,7 @@ export default defineConfig({
 		},
 		setupFiles: ['./vitest.setup.js'],
 		globals: true,
-		dangerouslyIgnoreUnhandledErrors: true,
+		// dangerouslyIgnoreUnhandledErrors: true,
```

Comment:
This was part of enabling `parallel` runs but we shouldn't enable this if we're not using it